src/mongo/db/catalog/collection.cpp

```diff
@@ -1129,7 +1129,7 @@ class RecordStoreValidateAdaptor : public ValidateAdaptor {
 
                 if (indexEntryCount != 0) {
                     indexEntryCount--;
-                    dassert(indexEntryCount >= 0);
+                    //dassert(indexEntryCount >= 0);
```

Comment:
- Why did you comment out the dassert
- If the dassert should be removed, please just delete it, rather than commenting it out.